src/System.Text.Json/src/System/Text/Json/Writer/JsonWriterHelper.Escaping.cs

```diff
@@ -202,7 +201,7 @@ private static bool IsInRangeInclusive(uint value, uint lowerBound, uint upperBo
         private static bool IsLowWordSurrogate(uint @char)
             => (@char & 0xF800U) == 0xD800U;
 
-        public static SequenceValidity PeekFirstSequence(ReadOnlySpan<byte> data, out int numBytesConsumed, out Rune rune)
```

Comment:
Rune is only available in .NET Core 3.0. For simplicity, removing its use all together since we don't really rely on it heavily.

Reply:
Can you add a comment to that effect?